openfe/protocols/openmm_rfe/equil_rfe_settings.py

```diff
@@ -519,19 +424,35 @@ class RelativeHybridTopologyProtocolSettings(Settings):
     class Config:
         arbitrary_types_allowed = True
 
+    # Inherited things
+
+    forcefield_settings: OpenMMSystemGeneratorFFSettings
+    """Parameters to set up the force field with OpenMM Force Fields."""
+    thermo_settings: ThermoSettings
+    """Settings for thermodynamic parameters."""
+
```

Comment:
I take it there's no reasonable way to inherit these docstrings from parent classes? (Curious why, for my own understanding.)

Reply:
For `forcefield_settings`, the base class is meant to be generic over other force field generators. For `thermo_settings`, it could be placed in the base class with a PR to GUFE.